examples/catalog/android/app/build.gradle

```diff
@@ -45,6 +45,9 @@ android {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
+            minifyEnabled true
+            useProguard true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
 }
```

Comment:
Is there some way we can do this that doesn't involve developers having to edit their build.grade files?

Reply:
We could configure the setting from the Flutter plugin, so developers don't have to set this up.  However,  we do need to provide a way to edit the proguard rules in the file `proguard-rules.pro`, in case  a plugin needs special rules.